python/img_hdf5_reader.py

```diff
@@ -197,12 +209,7 @@ def to_ometiff(self, channel_clips, sample, json_file):
 
     reader = ImgHdf5Reader(args.hdf5)
 
-    reader.to_pngs(
-        channel_clips=channel_clips,
-        sample=args.sample,
-        json_file=args.json_file
-    )
-
+    # Using instead of PNG functions
```

Comment:
In the future, It won't matter that we used to use PNG, so this comment won't be too useful?